api/server.js

```diff
@@ -20,18 +21,19 @@ manifestPromise.then(manifest => {
     server.auth.strategy('oidc_session', 'oidc_session');
     server.auth.strategy('email_token', 'email_token', {
       findToken: async id => {
-        let token = await bookshelf.model('email_token')
-          .forge({ token: id })
-          .where('expires_at', '>', bookshelf.knex.fn.now())
-          .fetch();
-
-        if (!token) {
-          token = await bookshelf.model('user_password_reset_token')
```

Comment:
I'm gonna do a quick check through the codebase to make sure that all the user_password_reset_token stuff is really gone (i got rid of a lot of it, but this was hanging around on purpose and I want to make sure that nothing's going to break by removing it)

Reply:
this checks out 👍 to removing this whole `if` block